src/components/DropZone/DropZone.tsx

```diff
@@ -224,26 +230,27 @@ export class DropZone extends React.Component<CombinedProps, State> {
       styles.DropZone,
       outline && styles.hasOutline,
       (active || dragging) && styles.isDragging,
+      loading && styles.isLoading,
       error && styles.hasError,
-      size && size === 'extraLarge' && styles.sizeExtraLarge,
-      size && size === 'large' && styles.sizeLarge,
-      size && size === 'medium' && styles.sizeMedium,
-      size && size === 'small' && styles.sizeSmall,
+      height && height === Size.ExtraLarge && styles.sizeExtraLarge,
```

Comment:
I don't think we need `height &&`, height will always be truthy (I think?) but it was undefined we still should be able to compare it the enum